src/components/Card.js

```diff
@@ -2,4 +2,6 @@
 // renders children inside div wrapper with className 'Card'
 // if there is no children component should return null;
 
-export default function Card() {}
\ No newline at end of file
+export default function Card({ children }) {
+	return <div className="Card">{children}</div>;
```

Comment:
Conditional rendering requirement is missing, 
before returning anything there must be a check if children exists or not 

```
export default function Card({ children }) {
       // early exit
        if (!children) return null;
	
        return <div className="Card">{children}</div>;
}
```